sdk/communication/azure-communication-jobrouter/README.md

```diff
@@ -195,7 +195,7 @@ from azure.eventgrid import EventGridEvent
 import json
 
 ## deserialize payload into a list of typed Events
-events = [EventGridEvent.from_json(json.loads(msg)) for msg in payload]
+events = [EventGridEvent.from_json(json.dumps(msg)) for msg in payload]
```

Comment:
```suggestion
events = [EventGridEvent.from_json(msg) for msg in payload]
```

There is a lack of clarity into what `msg` is coming from, so in doubt we should keep it simple. Also, `from_json` will expect a string, or a commplex type from store/eventhub